api/capi/src/nntrainer.cpp

```diff
@@ -782,6 +745,16 @@ int ml_train_layer_set_property(ml_train_layer_h layer, ...) {
   return status;
 }
 
+int ml_train_layer_set_property_with_single_param(ml_train_layer_h layer,
+                                                  const char *single_param) {
+  ML_TRAIN_VERIFY_VALID_HANDLE(layer);
+
+  if (single_param)
+    return ml_train_layer_set_property(layer, single_param, NULL);
```

Comment:
Is it needed this line?

If `single_param == NULL`, 
then `ml_train_layer_set_property(layer, NULL, NULL);`

so, I think it's enough with:
```
return ml_train_layer_set_property(layer, single_param, NULL);
```

Reply:
thank you, I will fix